OneCommunityGlobal/HighestGoodNetworkApp
# Shihao - Lint Fix Header

# Description
Lint Fix `src/component/Header/**`

## Main changes explained:
- Change the code style according to ESLint and Prettier

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as any user
4. go to dashboard
5. check the app works as normal (specific the `Header`)
6. set up ESLint and Prettier in VSCode
7. check all files in `src/component/Header/**` doesn't have errors

## Screenshots or videos of changes:
![Screenshot 2023-08-30 at 15 06 17 (2)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/0d7ae809-9132-4118-8a9d-0f4a14ded6f2)
